docs/src/use/configure/migration-guide.md

```diff
@@ -255,6 +255,63 @@ export default [
 ];
 ```
 
+In the eslintrc config system it was possible to use `eslint-env` configuration comments to define globals for a file.
+These comments are no longer recognized when linting with flat config: in a future version of ESLint, `eslint-env` comments will be reported as errors.
```

Comment:
Not sure about this part. [This issue](https://github.com/eslint/eslint/issues/13481) suggests that `eslint-env` config comments will start to _throw_ errors (in Phase 5: Remove `eslintrc`), but I guess the actual intent was to have these comments reported as problems by the linter? The [RFC](https://github.com/eslint/rfcs/blob/main/designs/2019-config-simplification/README.md?plain=1#L872) only mentions that `eslint-env` comments should be ignored (this is the current behavior).

Reply:
> Yes, the intent was that these would be reported as linter warnings.

Thanks! I've updated the wording in issue #13481 to clarify.

Reply:
I'll also try adding this next as I'm sure this will be helpful for people as they transition to the new config system.